File: flatseeker-backend/src/main/java/com/yakzhanov/flatseeker/model/BathroomStatus.java
package com.yakzhanov.flatseeker.model;

import lombok.Getter;

public enum BathroomStatus {

    SINGLE ("Одна"),
    DOUBLE ("Две"),
    SINGLE_NO_SHOWER ("Одна (без ванной)"),
    DOUBLE_NO_SHOWER ("Две (без ванной)");

    @Getter
    private final String title;

    BathroomStatus(String title) {
        this.title = title;
    }
}

File: flatseeker-ui/src/app/model/dict/BathroomStatus.ts
export class BathroomStatus {
  public key: string;
  public title: string;

  constructor(key: string, title: string) {
    this.key = key;
    this.title = title;
  }

  public static mock() {
    return new BathroomStatus("", "");
  }
}
